### DISCUSSION TITLE
Song preview button in playlist room's beatmap card

### DISCUSSION THREAD

**Discussion by Ladwick** [ppy/osu#28126] [Ideas] [answered]
In case title is not descriptive enough, **idea description:**

I usually lurk in playlist rooms to find new cool beatmaps and I always listen to the preview of the song before downloading, so I have to get in the beatmap page, listen to the preview and download the beatmap from there, so maybe putting an additional preview button in the beatmap card on the playlist would be nice? Though this is kind of personal since I dunno if it's a common thing for people to go through this process to download beatmaps this way or if most people just hit the download without listening to the preview, I'll leave the idea here in case someone find it nice. I don't mention multiplayer rooms because there is kind of mandatory to download the map.

Silly representation of how this looks in my mind:
![image](https://github.com/ppy/osu/assets/37979533/ef5c6209-d094-4e0e-abc1-7357622568fa)
![image](https://github.com/ppy/osu/assets/37979533/a497ccc2-02b5-46c2-82ad-233193508922)

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/21315. Please upvote that original discussion (with more upvotes) to show your support!

**Reply by Ladwick** [reply]
Sorry for the duplicate, I'll do that!
